sources/packages/web/src/components/ReadMoreContainer.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div>
+    <slot></slot>
+    <div id="extendedContent" v-show="isExtended">
+      <slot name="extended"></slot>
+    </div>
+    <a id="toggleButton" @click.prevent="toggleExtended" href="#">{{
+      extendedLabel
+    }}</a>
+  </div>
+</template>
+
+<script lang="ts">
+import { ref } from "vue";
+export default {
+  props: {
+    collapsedText: {
+      type: String,
+      default: "Read More",
+      required: true,
+    },
+    expandedText: {
+      type: String,
+      default: "Read Less",
+      required: true,
+    },
+  },
+  setup(props: any) {
+    const isExtended = ref(false);
+    const extendedLabel = ref(props.collapsedText);
```

Comment:
Can you please add a comment here 
By Default the long text will be collapsed with the link option of "ReadMore "

Reply:
Added some comments.

Reply:
Where are the comments, i dont see them in the file